test/command_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Gnat.CommandTest do
+  use ExUnit.Case, async: true
+  alias Gnat.Command
+
+  test "formatting a simple unsub message" do
+    command = Command.build(:unsub, 12, []) |> IO.iodata_to_binary
+    assert command == "UNSUB 12\r\n"
+  end
+
+  test "formatting an unsub message with max messages" do
+    command = Command.build(:unsub, 12, [max_messages: 3]) |> IO.iodata_to_binary
```

Comment:
Your use if iodata_to_binary suggests that "iodata" and arrays (or lists in elixir parlance!) are the same thing. Is that true?